### ISSUE TITLE
get_avatar_url on linked_oauth2 provider is not returning an url anymore

### ISSUE THREAD

**Issue by bastbnl** [pennersr/django-allauth#2228]
`get_avatar_url` returns `None` while there is a profile picture. I guess it's related to v2 of the OAuth2 provider.

Adding a PR that is intended to fix the issue. This PR depends on the notes found in this line:
https://github.com/pennersr/django-allauth/blob/817b264d8de17ee4ff5bf7c12dccbb8b28f968cc/allauth/socialaccount/providers/linkedin_oauth2/provider.py#L77

Note that this needs settings:
```
    'linkedin_oauth2': {
        'PROFILE_FIELDS': [
            'id',
            'firstName',
            'lastName',
            'profilePicture(displayImage~:playableStreams)',
        ],
        'PROFILEPICTURE': {
            'display_size_w_h': (400, 400.0),
        },
    },

```

**Comment by pennersr** [comment]
Closed via 1dfd3cc0

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/2228
